src/deploy/project.py

```diff
@@ -239,8 +239,14 @@ def is_release_deployed(self, release, environment_id):
                 deployed_tasks = []
                 for task in tasks:
                     task_tags = parse_aws_tags(task["tags"])
-                    task_deployment_label = task_tags["deployment:label"]
-                    deployed_tasks.append(task_deployment_label == service_deployment_label)
+
+                    _is_task_deployed = (
+                        task_tags.get("deployment:label") == service_deployment_label
```

Comment:
`task_tags.get` will return `None` if the dict member is not available - rather than throwing an error. `deployment:label` may not be present when moving from unmanaged tasks to managed tasks with `weco-deploy`.